### ISSUE TITLE
weight decay and learning rate in convolution layer.

### ISSUE THREAD

**Issue by cheer37** [vlfeat/matconvnet#411]
I saw to describe the weight decay and learning rate in convolution layer.
But simplenn wrapper doesn't use it when calling mex function.
weight decay and learning rate in convolution layer can work for only in Dag?

**Comment by usmankhalid-developers** [comment]
They also wok in simple. 
Example code

% Define network
net.layers = {} ;

net.layers{end+1} = struct('type', 'conv', ...
    'weights', {{sqrt(2/(9*64))*randn(3,3,1,64,'single'), zeros(64,1,'single')}}, ...
    'stride', 1, ...
    'pad', 1, ...
    'dilate',1, ...
    'learningRate',[1 1], ...
    'weightDecay',[1 0], ...
    'opts',{{}}) ;
net.layers{end+1} = struct('type', 'relu','leak',0) ;
